src/main/java/com/google/research/bleth/simulator/Board.java

```diff
@@ -0,0 +1,58 @@
+package com.google.research.bleth.simulator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/** A container for all the agents, representing their locations, either real or estimated. */
+public class Board {
+    public final int rowNum;
```

Comment:
Avoid exposing internal fields publicly - prefer to provide accessors for them.
(Accessors enable you to change implementation details without affecting your callers - e.g., in this case, if you decide to accept my suggestion to use a Table, even the test should not know)